command/title_body_survey.go

```diff
@@ -88,39 +125,38 @@ func selectTemplate(templatePaths []string) (string, error) {
 	return string(templateContents), nil
 }
 
-func titleBodySurvey(cmd *cobra.Command, providedTitle, providedBody string, defs defaults, templatePaths []string, allowPreview bool) (*titleBody, error) {
+func titleBodySurvey(cmd *cobra.Command, data *titleBody, apiClient *api.Client, repo ghrepo.Interface, providedTitle, providedBody string, defs defaults, templatePaths []string, allowReviewers bool) error {
 	editorCommand := os.Getenv("GH_EDITOR")
 	if editorCommand == "" {
 		ctx := contextForCommand(cmd)
 		cfg, err := ctx.Config()
 		if err != nil {
-			return nil, fmt.Errorf("could not read config: %w", err)
+			return fmt.Errorf("could not read config: %w", err)
 		}
 		editorCommand, _ = cfg.Get(defaultHostname, "editor")
 	}
 
-	var inProgress titleBody
-	inProgress.Title = defs.Title
+	data.Title = defs.Title
 	templateContents := ""
 
 	if providedBody == "" {
 		if len(templatePaths) > 0 {
 			var err error
 			templateContents, err = selectTemplate(templatePaths)
 			if err != nil {
-				return nil, err
+				return err
 			}
-			inProgress.Body = templateContents
+			data.Body = templateContents
```

Comment:
It took me a second to figure out what data is used for. Is it a go idiom to use the name data for a var that returns a value?

Reply:
I agree that `data` isn't the best variable name. Basically, `data` here is an instance of `titleBody` which by now basically serves as a state object during the wizard. I'll try to rename these variables and types so it's more clear what's going on.